typescript/cli/src/deploy/warp.ts

```diff
@@ -544,55 +499,46 @@ export async function runWarpRouteApply(
       context.registry,
     );
 
-  const transactions: AnnotatedEV5Transaction[] = [
-    ...(await extendWarpRoute(
-      params,
-      apiKeys,
-      warpDeployConfig,
-      warpCoreConfigByChain,
-    )),
-    ...(await updateExistingWarpRoute(
-      params,
-      apiKeys,
-      warpDeployConfig,
-      warpCoreConfigByChain,
-    )),
-  ];
+  // Extend the warp route and get the updated configs
+  const updatedWarpCoreConfig = await extendWarpRoute(
+    params,
+    apiKeys,
+    warpCoreConfig,
+  );
+
+  // Then create and submit update transactions
+  const transactions: AnnotatedEV5Transaction[] = await updateExistingWarpRoute(
+    params,
+    apiKeys,
+    warpDeployConfig,
+    updatedWarpCoreConfig,
+  );
```

Comment:
It seems that when enrolling new routers, the addresses are added correctly to the routers but, the destination gas is set to 0 instead of the default value for the type of warp route

```yaml
anvil420:
  decimals: 18
  destinationGas:
    "42069": "0"
    "69": "0"
  ...
```